readme.md

```diff
@@ -160,6 +243,8 @@ Once installation is working, take a look at the existing code to make sure you
 
 * **Your Answer:**
 
+TODO
```

Comment:
-The first argument `payload` represents the data to be passed in the JWT, 
The second argument is the secret, in this case `MYSECRETPASSCODE`
The last argument `options` can include a number of different options such as expiresIn, algorithm, audience, issuer, etc.